responsibleai/responsibleai/managers/counterfactual_manager.py

```diff
@@ -577,6 +577,67 @@ def compute(self):
                     cf_config.failure_reason = str(e)
                     raise e
 
+    def request_counterfactuals(self, query_id: str, data: Any):
+        """Return the counterfactuals for a given point.
+
+        :param query_id: The query id for the finding the
+                         counterfactual config.
+        :type query_id: str
+        :param data: The data point for which the counterfactuals
+                     need to be generated.
+        :type data: Any
+        :return: The serialized version counterfactuals for the
```

Comment:
this description is a bit confusing, shouldn't it be the serialized counterfactuals as a dictionary, and not a version, or did I misunderstand?

Reply:
Thanks for pointing this out. Changed to following:- 

        :return: An object of type CounterfactualData with
                 counterfactuals for the given data point.
        :rtype: CounterfactualData

Reply:
I made the API return object of type CounterfactualData. Thanks for pointing this out.